build-system/tasks/check-types.js

```diff
@@ -283,13 +283,20 @@ async function typeCheck(targetName) {
     return;
   }
 
+  if (target.onError) {
+    // If an onError handler is defined, steal the output and log it manually.
+    opts.logger = (m) => {
+      errorMsg = m;
+      log(m);
```

Comment:
We probably should not be calling `log(m)` here. The `onError` fn should be in charge of determining output (we want it swallowed).

Reply:
You're right, removed.